### DISCUSSION TITLE
I'd like some really basic theming (color) options

### DISCUSSION THREAD

**Discussion by StachuDotNet** [darklang/dark#4395] [Feedback]
At minimum, I want a 'light mode' that I can toggle between.

My specific use case is that I often screen-shot Dark and send the screenshot to my eInk tablet to later review the UI and raw over it (sketching). The current Dark background doesn't render well on it.

If this is is unlikely to be supported officially, I'll probably write a small bookmarklet to run JS that does this for me soon.

**Comment by StachuDotNet** [comment]
IIRC, there's a branch of old code somewhere that included efforts towards this.

**Comment by pbiggar** [comment]
I'd like to support this, but probably in a reasonable distant future, as this will cost us a lot of work for each new feature, so this probably only makes sense to add once there isn't so much to change
